File: engine/model/src/main/scala/net/server/channel/packet/cash/UseSpResetPacket.scala
package net.server.channel.packet.cash

class UseSpResetPacket(private var _position: Short, private var _itemId: Int, private var _to: Int, private var _from: Int) extends AbstractUseCashItemPacket(_position, _itemId) {
  def to: Int = _to

  def from: Int = _from
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/use/UseSpResetPacket.java
package net.server.channel.packet.cash.use;

public class UseSpResetPacket extends AbstractUseCashItemPacket {
   private final Integer to;

   private final Integer from;

   public UseSpResetPacket(Short position, Integer itemId, Integer to, Integer from) {
      super(position, itemId);
      this.to = to;
      this.from = from;
   }

   public Integer to() {
      return to;
   }

   public Integer from() {
      return from;
   }
}
